int32 robot_id
int32 trans_kind
RouteSP goal
